src/Cedar/Console.c

```diff
@@ -2048,7 +2048,13 @@ bool PasswordPrompt(char *password, UINT size)
 		else if (c == 0xE0)
 		{
 			// Read one more character
+
+#ifdef	OS_WIN32
 			c = getch();
+#else	// OS_WIN32
+			c = getc(stdin);

```

Comment:
Which problem does this line solve???

Reply:
I find the C lib reference, getch() is not a stdio.h standard function, also got "implicit declaration" error.

. On the other hand the original author using a conditional compile to solve it in the upper context. so I copy and paste

please see the old file line 2019~2023